lighthouse-core/gather/gatherers/accessibility.js

```diff
@@ -45,6 +45,7 @@ function runA11yChecks() {
     // Augment the node objects with outerHTML snippet & custom path string
     // @ts-ignore
     axeResult.violations.forEach(v => v.nodes.forEach(node => {
+      // @ts-ignore - getNodePath put into scope via stringification
       node.path = getNodePath(node.element);
```

Comment:
you could keep this the same, if you also export `getNodePath` in `page-functions.js`.

Will need that module to export `getNodePath` for linking nodes in the report for the font-size audit (#6436) (and for other non-accessibility audits).

Reply:
I'm not sure what you mean @Hoten, it's run in the context of the page and he's exported the stringified version which is what he'll need, no?

Reply:
Didn't realize all of that was running in context of the page. Nevermind!